### ISSUE TITLE
Could not find symbol INST_CONSTANT_874439

### ISSUE THREAD

**Issue by rodrigo7491** [cvc5/ethos#49]
When trying to check the AletheLF proof produced for the attached .smt2 file (see [test.zip](https://github.com/user-attachments/files/15848499/test.zip)) I get the following error:
```
Error: test.proof:266573.25: Could not find symbol INST_CONSTANT_874439
```
For the same input, I could produce and successfully check a LFSC proof.

Following the discussion on #36, and given that the message refers to a symbol rather than a rule, I believe the problem should not be on my setup. I'd be curious to know if someone else could reproduce this behaviour.

**Comment by ajreynol** [comment]
I believe this is a bug in cvc5. I will look into a fix soon.

**Comment by ajreynol** [comment]
I tried this on cvc5 `994c86b5dc4cc6d3a54f9fc1cdda593fd9372012` and couldn't reproduce. 

What version of cvc5 are you using? Also, what command line options are you using exactly?

**Comment by rodrigo7491** [comment]
I've been using the latest release of cvc5, v1.1.2, so the commit should be `435f2d44f1bb37c4c17d2b59615b7323039b7a62`. This is also the commit I got the alfc signatures from. The alfc version I've been running is from commit `eca2cbd4e27712c35a9111a3c9c517f49c593500`.

The CLI commands I used were:

* `cvc5 --proof-format=alf --proof-granularity=theory-rewrite file.smt2`
*  `alfc file.proof`
 
 The smt2 file already has `(set-option :produce-proofs true)` and `(get-proof)`, so no need for the `--dump-proofs` option. In the proof file I manually changed `unsat` in the first line to `(include "alf/cvc5/Cvc5.smt3")`.
